### ISSUE TITLE
is it possible to place missing values last when doing a descending hashsort?

### ISSUE THREAD

**Issue by ractingmatrix** [mcaceresb/stata-gtools#31]
currently the default seems to place missing values first.

**Comment by ractingmatrix** [comment]
Raised this issue because I was expecting the same outcome from gsort and hashsort. 
Thanks for the suggestion.

**Comment by mcaceresb** [comment]
You are correct. I have corrected the note hashsort prints to reflect this.

gsort treats missing values sepparately as a group, but sorts missing values accordingly within a missing group (i.e. extended missing values appear last as a group but are sorted in descending order). When I first implemented hashsort I found this behavior strange, so I coded hashsort to work as if "mfirst" was always passed.

I will work on either adding an "mlast" option or redefining the behavior of hashsort. Thank you for pointing this out.
